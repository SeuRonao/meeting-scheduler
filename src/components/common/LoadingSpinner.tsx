import React from "react";
import { Spinner } from "react-bootstrap";
import { useTranslation } from "react-i18next";

export default function LoadingSpinner({ className }: { className?: string }) {
  const { t } = useTranslation();
  return (
    <Spinner className={className} animation="border" role="status">
      <span className="visually-hidden">{t("LoadingSpinner.loading")}</span>
    </Spinner>
  );
}
