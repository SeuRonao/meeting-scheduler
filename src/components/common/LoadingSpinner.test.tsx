import renderer from "react-test-renderer";
import LoadingSpinner from "./LoadingSpinner";

describe("Testing LoadingSpinner", () => {
  test("renders correctly", () => {
    const component = renderer.create(<LoadingSpinner />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
