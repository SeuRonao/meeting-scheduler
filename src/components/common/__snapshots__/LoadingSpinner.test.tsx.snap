// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing LoadingSpinner renders correctly 1`] = `
<div
  className="spinner-border"
  role="status"
>
  <span
    className="visually-hidden"
  >
    loading
  </span>
</div>
`;
